examples/ts/create-order-position-with-takeprofit-stoploss.ts

```diff
@@ -0,0 +1,86 @@
+// @ts-nocheck
```

Comment:
@ttodua the examples are not 100% equal, can we keep both for now?

Reply:
I think more files for same example would be unnecessary and would not benefitable for users, but oky as you wish, reverted & pushed.

Reply:
@ttodua it's okay as long as they're both correct

Reply:
i think now it's ready